packages/desktop/src/renderer/app/components/environment-routes/environment-routes.component.html

```diff
@@ -447,6 +447,18 @@
             />
           </div>
 
+          <ng-container [formGroup]="activeRouteResponseForm">
+            <div class="input-group pt-2 px-2 col-6">
+              <input
+                *ngIf="activeRoute.type === 'crud'"
+                type="text"
+                class="form-control"
+                formControlName="crudKey"
+                placeholder="Add the index for crud operations"
```

Comment:
I am not sure "index" is clear enough. json-server call it an "id property". I think it's a bit clearer. 
What do you think? 

It could be changed to: "id property for crud operations ("id", "uuid", etc.)"